src/platforms/android/androidglean.cpp

```diff
@@ -0,0 +1,138 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "androidglean.h"
+#include "androidutils.h"
+#include "constants.h"
+#include "leakdetector.h"
+#include "logger.h"
+#include "mozillavpn.h"
+#include "settingsholder.h"
+#include "jni.h"
+#include "mozillavpn.h"
+#include "qmlengineholder.h"
+#include "androidvpnactivity.h"
+
+#include <QApplication>
+#include <QJsonDocument>
+#include <QJsonObject>
+#include <QJniObject>
+#include <QJniEnvironment>
+#include <QQmlApplicationEngine>
+#include <QJSEngine>
+#include <QQmlEngine>
+#include <QFileInfo>
+#include <QFile>
+
+namespace {
+AndroidGlean* s_instance = nullptr;
+Logger logger(LOG_ANDROID, "AndroidGlean");
+
+}  // namespace
+
+// static
+void AndroidGlean::initialize(QQmlEngine* e) {
+  if (!s_instance) {
+    Q_ASSERT(qApp);
+    s_instance = new AndroidGlean(qApp);
+  }
+  auto glean_db_path = e->offlineStorageDatabaseFilePath("Glean") + ".sqlite";
+  auto gleanDB = QFileInfo(glean_db_path);
+  if (gleanDB.exists()) {
+    QFile::remove(glean_db_path);
```

Comment:
why do we remove it?

Reply:
As that is only used by glean.js, it's just wasting disk space for users who upgrade from 2.8 -> 2.9

Reply:
OK write a comment.